client/src/pages/EpisodeShowPage/Timer/index.tsx

```diff
@@ -0,0 +1,127 @@
+import * as _ from "lodash";
+import * as moment from "moment";
+import * as React from "react";
+import { Glyphicon } from "react-bootstrap";
+
+class TimerPresentational extends React.Component<any,  any> {
+  public render() {
+    const timerRunning = !(this.props.timeLeft === Infinity || this.props.timeLeft <= 0);
+
+    return (
+      timerRunning
+      &&
+      <span>
+        <span
+          style={{
+            color: "#137a9a",
+            fontVariant: "small-caps",
+            fontWeight: 700,
+          }}
+        >
+          <Glyphicon glyph="time" /> time left
+        </span>
+          <br />
+        <span
+          style={{
+            fontSize: "28px",
+          }}
+        >
+          {moment(this.props.timeLeft).format("m")}
+        </span>
+        m
+        {" "}
+        <span
+          style={{
+            fontSize: "28px",
+          }}
+        >
+          {moment(this.props.timeLeft).format("ss")}
+        </span>
+        s
+      </span>
```

Comment:
I suggest writing a function with string interpolation to do the above,  if moment JS can't do it. 

let convert(time) = `${}m ${s}s`...